src/axis.js

```diff
@@ -172,3 +171,22 @@ export function axisBottom(scale) {
 export function axisLeft(scale) {
   return axis(left, scale);
 }
+
+function maybeMultilineText(format, orient, x, space) {
+  const repeats = [];
+  const lines = [];
+  return (text) => {
+    text
+    .attr(x, space)
+    .text((d, i) => {
+      const cur = format(d) || "";
```

Comment:
I expect this needs to coerce the result of format to a string, and we should only map null and undefined to the empty string, not other falsey values such as `0` and `false`.

Reply:
done